integration_tests/ads/test_ad_groups.py

```diff
@@ -63,9 +65,7 @@ def test_update_success(self):
         )
 
         new_name = "SDK_AD_GROUP_NEW_NAME"
-        new_spec = {
-                "GENDER": ["male"]
-        }
+        new_spec = TargetingSpec(gender=["male"])
```

Comment:
We shouldn't be using models directly like this from the generated client. we need a copy of its implementation in the SDK and then use that if needed. Was there an issue with just using what we had earlier?